source/docs/advanced/graphql/remote-schemas.md

```diff
@@ -0,0 +1,191 @@
+module.exports = {
+-  namespace: "Pokemon"
++  namespace: "Pokemon",
++  remoteSchema: {
++    uri: "https://graphql-pokemon.now.sh/"
++  }
+};
+```
+
+**Restart your application** and explore your schema using the GraphQL playground (by default at
+[http://localhost:4000/playground](http://localhost:4000/playground)).
+
+You must see several new top level queries prefixed by `Pokemon_`.
+**Congratulations!** You now have access to all the Pokemon remote GraphQL queries and mutations in your Front-Commerce application, alongside your existing eCommerce ones.
+
+Try the query below for instance:
+
+```graphql
+{
+  Pokemon_pokemon(name:"raichu"){
+    name
+    weight { minimum maximum }
+    image
+    classification
+  }
+
+  product(sku:"VD11") {
+    name
+  }
+}
+```
+
+## Default transforms
+
+You may have noticed in the previous example, that the `pokemon` query was named `Pokemon_pokemon`.
+The reason is that Front-Commerce applies some transforms to the remote schema before merging it into the local schema.
+The goal is to allow to quickly merge remote schemas by reducing the probability of a conflict.
+
+Here are the transforms applied to the remote schema by default.
+
+### Prefix root queries and mutations
+
+Root queries and mutations are prefixed with the GraphQL module namespace and an underscore.
+In the example above, the remote query `pokemon(name: String!)` was renamed into `Pokemon_pokemon(name: String!)`.
+If we renamed the module’s namespace in `MyPokedex` the query would be `MyPokedex_pokemon(name: String!)`.
+
+<blockquote class="important">
+  **Important:** _special_ characters are stripped from the module namespace to obtain a valid name.
+  Example: for a module namespaced `Acme/Foo`, Front-Commerce will use `AcmeFoo_` as a prefix.
+</blockquote>
+
+### Prefix types
+
+In a similar way, types from the remote schema are prefixed with the GraphQL module namespace and an underscore.
+
+In the Pokemon example above schema, the `PokemonDimension` type would be renamed to `Pokemon_PokemonDimension`.
+
+## Apply custom transforms
+
+When doing remote schema stitching, you may want to apply your own transforms to the remote schema.
+
+For instance, you may want to only expose the `pokemons` query from the remote schema.
+Here is how you could achieve it thanks to Front-Commerce’s [`remoteSchema.transforms` GraphQL module definition key](/docs/reference/graphql-module-definition.html#transforms-optional):
+
+```diff
++ const { FilterRootFields } = require("graphql-tools");
++
+module.exports = {
+  namespace: "Pokemon",
+  remoteSchema: {
+-    uri: "https://graphql-pokemon.now.sh/"
++    uri: "https://graphql-pokemon.now.sh/",
++    transforms: [
++      new FilterRootFields(
++        (operation, rootField) =>
++          operation === "Query" && rootField === "pokemons"
++      )
++    ]
+  }
+};
+```
+
+<blockquote class="info">
+  Please note that these transforms are applied **before** Front-Commerce default ones.
```

Comment:
Does this mean that the query will still be named `Pokemon_pokemons` ?
Whether it's true or false, it should be make explicit here in my opinion.